File: hydrotech/hydrotech/report/ht_quotation_report/ht_quotation_report.js
// Copyright (c) 2016, Taazur and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["HT Quotation Report"] = {
	"filters": [

    {
    "fieldname":"from_date",
    "label" :("From Date"),
    "fieldtype":"Date",
    "default":frappe.datetime.month_start(date)
    },

    {
    "fieldname":"to_date",
    "label":("To Date"),
    "fieldtype":"Date",
    "default":get_today()
    }

	]
};

File: hydrotech/hydrotech/report/ht_quotation_report/ht_quotation_report.py
# Copyright (c) 2013, Taazur and contributors
# For license information, please see license.txt

from __future__ import unicode_literals
# import frappe

from __future__ import unicode_literals
import frappe
from frappe import _
from frappe.utils import flt, getdate

def execute(filters=None):
	columns, data = [], []
	columns=get_columns()
	data=get_data(filters)
        return columns, data

def get_columns():
	return [
                _("Quotation") + ":Link/Quotation:100",
                _("Customer ID") + ":Link/Customer:100",
                _("Customer Name") + ":Link/Customer:130"
        ]
